### DISCUSSION TITLE
How do I check if the database is unlocked?

### DISCUSSION THREAD

**Discussion by Flinner** [keepassxreboot/keepassxc#7906] [Q&A] [answered]
is there a way to check if keepassxc is ulocked from the command line?

```bash
keepassxc --status
```
or something like that

I want to write a polybar module that indicates if the database is currently open or not.

thanks for this amazing software

**Comment by Flinner** [comment]
I got until now:

use secret tool to store anything, and then check if i can access it

```sh
❯ secret-tool store --label='is_keepassxc_open' check status
Password: yes

# use this to check everytime
❯ secret-tool lookup check status
yes # databse is unlocked

```

**Comment by droidmonkey** [comment, marked as answer]
No we do not disclose that information that easily. You will have to use the browser api and register a connection.

**Reply by Flinner** [reply]
thanks for the fast reply! I managed to use secret-tool to access the api

(the idea poped in my head just after asking the quesion 😅, even though i was searching for something in  a long time)

**Reply by droidmonkey** [reply]
Clever trick 😁
